modules/renter/hostdb/hostdb.go

```diff
@@ -230,6 +230,18 @@ func (hdb *HostDB) Host(spk types.SiaPublicKey) (modules.HostDBEntry, bool) {
 	return host, exists
 }
 
+// InitialScanComplete returns a boolean indicating if the initial scan of the
+// hostdb is completed.
+func (hdb *HostDB) InitialScanComplete() (complete bool, err error) {
```

Comment:
Should we have this be a separate function entirely, or just an extra return value in `QueuedScans()`? Might be simpler to have just one function which does both things. I could go either way.

Reply:
Yep also something I thought about when writing it. I like to keep things separated but that's just my preference.